doc/user/content/release-notes.md

```diff
@@ -46,6 +46,8 @@ Use relative links (/path/to/doc), not absolute links
 Wrap your release notes at the 80 character mark.
 {{< /comment >}}
 
+- Support gzip compressed file sources
```

Comment:
- include link to the documentation where someone can find more info (`/sql/create-source/text-file/#compression`)
- include link to this PR for reference to the work done, i.e. `{{% gh 5392 %}}`
- s/gzip compressed/`gzip`-compressed/
- terminate sentence with period